cmake/gcovr.cmake

```diff
@@ -0,0 +1,16 @@
+if(USE_GCOV)
+ 
+    SET(GCOVR gcovr)
```

Comment:
Here we are assuming the existence of `gcovr`. Please, use `find_program` to check whether it exists:

https://cmake.org/cmake/help/v3.5/command/find_program.html